### DISCUSSION TITLE
Cpu usage so high how to reduce?

### DISCUSSION THREAD

**Discussion by Arvin2A** [dnschneid/crouton#4614] [Q&A]
I just retried xfce 4.16 in xenial and when I ran chromium or any web browser my gpu is OK but everything so so SLOW

**Comment by ghost** [comment]
Are there other things running in the background (outside or inside of your chroot)? 
What other web browsers have your tried? Could you also try another application and see how that runs? 
What CPU do you have is it older, or just slower in general?

**Reply by Arvin2A** [reply]
well i have an intel celeron N4000 and i had just crosh tab and just chromium in. I used the default netsurf but it works only 20% better than chromium

**Reply by sanrobin** [reply]
I also do have Celeron N4000 but I never noticed high CPU Usage. How much RAM do you have? Also Chrome is a memory resource eater! Even Crosh tab is equal to a Chrome tab. Maybe try -b. It may show some increased performance.

EDIT: -b means the chroot to run in background, therefore having an option to close crosh.
